### ISSUE TITLE
Attribute Upcoming_waste_types of sensor.acv_eerstvolgende isn't available anymore

### ISSUE THREAD

**Issue by RWSF74** [pippyn/Home-Assistant-Sensor-Afvalbeheer#537]
The attribute of **sensor.acv_eerstvolgende**, **Upcoming_waste_types** isn't available anymore in my config:
<img width="1046" alt="Schermafbeelding 2024-12-06 172317" src="https://github.com/user-attachments/assets/155cf8b3-d008-4740-b54c-330ab59305a3">

I used this attribute to be able to show it's text in a one-button tile, via a (custom) sensor:
`# sensor afvalbeheer
  - sensor:
      - name: afval_ACV_eerstvolgend_type
        state: "{{ state_attr('sensor.acv_eerstvolgende','Upcoming_waste_types') }}"`

Can this be fixed, or do you have an alternative? Thanks in advance!
PS i noticed this after last update, but im not sure if it was not caused by an earlier update...

**Comment by pippyn** [comment]
can you share your config? with an address for me to test with?

**Comment by GroteGehaktBal** [comment]
Yes this is my config:

```
afvalbeheer:
  wastecollector: offalkalinder
  resources:
    - restafval
    - gft
    - papier
    - textiel
  postcode: 9851AD
  streetnumber: 70
  upcomingsensor: 1                # (optional)
  dateformat: '%d %b'              # (optional)
  dateonly: 0                      # (optional)
  dayofweekonly: 1                 # (optional)
  nameprefix: 1                    # (optional)
  builtinicons: 1                  # (optional)
  builtiniconsnew: 1               # (optional)
  dutch: 1                         # (optional)
  updateinterval: 6
  ```

**Comment by RWSF74** [comment]
This is my config:

```
afvalbeheer:
  wastecollector: ACV
  resources:
    - restafval
    - gft
    - papier
    - pmd
  postcode: 6713LG
  streetnumber: 143
  upcomingsensor: 1                # (optional)
  dateformat: '%a %d-%m'           # (optional)
  dateonly: 0                      # (optional)
  name: ""                         # (optional)
  nameprefix: 1                    # (optional)
  builtinicons: 1                  # (optional)
  dutch: 1                         # (optional)
```


And the sensor I used earlier:
```
# sensor afvalbeheer
  - sensor:
      - name: afval_ACV_eerstvolgend_type
        state: "{{ state_attr('sensor.acv_eerstvolgende','Upcoming_waste_types') }}"
```

**Comment by pippyn** [comment]
Fixed in v5.5.4
